src/components/IdeaForm.tsx

```
@@ -61,111 +66,172 @@ export default function IdeaForm() {
+              <TextareaAutosize
+                id="idea-input"
+                value={idea}
+                onChange={(e) => setIdea(e.target.value)}
+                placeholder="Describe your project idea in detail..."
+                className="input resize-none overflow-auto w-full p-2 border rounded h-full focus:outline-none focus:ring-2 focus:ring-blue-500 focus:border-transparent"
+                minRows={10}
+                required
+                aria-describedby="idea-input-description"
+              />
+              <div id="idea-input-description" className="text-sm text-gray-500">
+                Describe your project idea, target audience, and key features. The more detail you provide, the better suggestions you'll receive.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)